frappe/website/web_template/subnav/subnav.html

```diff
@@ -0,0 +1,21 @@
+<nav class="navbar navbar-expand-lg navbar-light subnav">
+	<div class="container">
+		<a class="navbar-brand d-none d-sm-block" href="{{ path }}">
+			{% if logo %}
+			<img class="d-inline-block align-top mr-2" src="{{ logo }}" style="max-height: 25px;"/>
+			{% endif %}
+			<span class="subnav-title">{{ title }}</span>
+		</a>
```

Comment:
Don't use Bootstrap's utility classes here because they use `!important` and difficult to override using Custom CSS.